x/poe/bootstrap_integration_test.go

```diff
@@ -127,3 +126,20 @@ func withRandomValidators(t *testing.T, ctx sdk.Context, example keeper.TestKeep
 		})
 	}, collectValidators
 }
+
+func membersAreAllVotingMembers(ctx sdk.Context, members []string, contractAddr sdk.AccAddress, tk types.TWasmKeeper) (bool, error) {
+	ocTrustedCircleAdapter := contract.NewTrustedCircleContractAdapter(contractAddr, tk, nil)
+	voters, err := ocTrustedCircleAdapter.QueryListVoters(ctx)
+	if err != nil {
+		return false, err
+	}
+	notVotingMembers := make(map[string]struct{}, len(members))
+	for _, m := range members {
+		notVotingMembers[m] = struct{}{}
+	}
+
+	for _, v := range voters.Members {
+		delete(notVotingMembers, v.Addr)
+	}
+	return len(notVotingMembers) == 0, nil
```

Comment:
There is still the case that the contract contains more members than you have passed. Please ensure that len(voters)==len(members)